src/sentry/deletions/defaults/project.py

```diff
@@ -47,7 +53,6 @@ def get_child_relations(self, instance):
             IncidentProject,
```

Comment:
I _think_ all the other related tables should be deleted on their own through foreign keys, though I'm curious why this one is in here since it has a foreign key to `project`.